qadence/execution.py

```diff
@@ -111,10 +111,7 @@ def sample(
     noise: Union[Noise, None] = None,
     configuration: Union[BackendConfiguration, dict, None] = None,
 ) -> list[Counter]:
-    """Convenience wrapper for the `QuantumModel.sample` method.  This is a
-    `functools.singledispatch`ed function, so it can be called with a number of different arguments.
-    See the examples of the [`expectation`][qadence.execution.expectation] function. This function
-    works exactly the same.
+    """Convenience wrapper for the `QuantumModel.sample` method.
```

Comment:
The same as above.

Reply:
Here too, compliant with the formatter. I am not sure the other lines about single dispatch are really informative as it is obvious from the code.